examples/notebooks/beam-ml/run_inference_basic.ipynb

```diff
@@ -1362,6 +1308,408 @@
+            "\u001b[K     |████████████████████████████████| 118 kB 45.3 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 234 kB 62.3 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 234 kB 58.1 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 234 kB 56.7 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 234 kB 66.2 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 234 kB 36.8 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 265 kB 56.4 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 148 kB 55.4 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 147 kB 65.9 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 4.6 MB 35.9 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 5.9 MB 43.6 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 1.1 MB 50.5 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 438 kB 49.5 MB/s \n",
+            "\u001b[K     |████████████████████████████████| 1.7 MB 48.0 MB/s \n",
+            "\u001b[?25h  Building wheel for dill (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
+            "  Building wheel for google-apitools (setup.py) ... \u001b[?25l\u001b[?25hdone\n",
+            "  Building wheel for docopt (setup.py) ... \u001b[?25l\u001b[?25hdone\n"
+          ]
+        }
+      ]
+    },
+    {
+      "cell_type": "markdown",
+      "source": [
+        "## Authenticate\n",
+        "\n",
+        "Authorize yourself so that you can save and load models."
+      ],
+      "metadata": {
+        "id": "8RHZxqB1HLlr"
+      }
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "from google.colab import auth\n",
+        "auth.authenticate_user()"
+      ],
+      "metadata": {
+        "id": "qDdGKYOoevNb"
+      },
+      "execution_count": 2,
+      "outputs": []
+    },
+    {
+      "cell_type": "code",
+      "source": [
+        "import argparse\n",
+        "\n",
+        "import tensorflow as tf\n",
+        "from tensorflow import keras\n",
+        "from tensorflow_serving.apis import prediction_log_pb2\n",
+        "\n",
+        "import apache_beam as beam\n",
+        "from apache_beam.ml.inference.base import RunInference\n",
+        "import tfx_bsl\n",
+        "from tfx_bsl.public.beam.run_inference import CreateModelHandler\n",
+        "from tfx_bsl.public import tfxio\n",
+        "from tfx_bsl.public.proto import model_spec_pb2\n",
+        "from tensorflow_metadata.proto.v0 import schema_pb2\n",
+        "\n",
+        "import numpy\n",
+        "\n",
+        "from typing import Dict, Text, Any, Tuple, List\n",
+        "\n",
+        "from apache_beam.options.pipeline_options import PipelineOptions\n",
+        "\n",
+        "project = \"<Your Project>\"\n",
```

Comment:
project and bucket are defined at the top as well

Reply:
removed.